compiler/rustc_lint/Cargo.toml

```diff
@@ -27,3 +27,12 @@ rustc_type_ir = { path = "../rustc_type_ir" }
 tracing = "0.1"
 unicode-security = "0.1.0"
 # tidy-alphabetical-end
+
+[features]
+# tidy-alphabetical-start
+rustc_use_parallel_compiler = [
+    'rustc_data_structures/rustc_use_parallel_compiler',
+    'rustc_errors/rustc_use_parallel_compiler',
+    'rustc_middle/rustc_use_parallel_compiler',
+]
+# tidy-alphabetical-end
```

Comment:
Disclosure: `x test tidy` doesn't seem to actually enforce alphabetical ordering of these sub-lists, so the alphabetical check only applies to the top-level features.